tests/src/test/java/io/streamnative/pulsar/handlers/mqtt/base/MQTTProtocolHandlerTestBase.java

```diff
@@ -238,36 +237,62 @@ public void stopBroker(int brokerIndex) throws Exception {
 
     protected void startBroker() throws Exception {
         for (int i = 0; i < brokerCount; i++) {
```

Comment:
Looks the broker count does not change, is it expected for this PR?

Reply:
Yes, I will add another patch for multi-broker and update here.